.repo/sample-list-generator/src/utils.ts

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs/promises';
```

Comment:
Can you please split this file up into more meaningful parts, util classes are an antipattern as they quickly grow and break separation of concerns.